app/assets/appfilter.xml

```diff
@@ -2336,6 +2336,7 @@
   <item component="ComponentInfo{com.thehitman7.cygisk/com.topjohnwu.magisk.ui.MainActivity}" drawable="magisk" name="Cygisk" />
   <item component="ComponentInfo{com.cyworld.camera/com.cyworld.cymera.CameraMain}" drawable="cymera" name="Cymera" />
   <item component="ComponentInfo{me.tigerhix.cytoid/me.tigerhix.cytoid.CytoidPluginActivity}" drawable="cytoid" name="Cytoid" />
+  <item component="ComponentInfo{PWA}" drawable="czynaczas" name="Czynaczas.pl" />
```

Comment:
Czynaczas.pl → Czynaczas

Reply:
The official app name used by the developer is `Czynaczas.pl`, as you can see in the about page here: https://czynaczas.pl/o-aplikacji (just search for `czynaczas`, and every time it's going to have `.pl` at the end)

Reply:
In general, this is not very important in this case. "pl" is easy to remove without losing meaning, because there is no "pl" on their logo, and this is just a domain. If you leave it, then ok.